Cargo.toml

```diff
@@ -212,7 +212,7 @@ atomic-waker = "1"
 js-sys = "0.3.64"
 wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
-web-time = "0.2"
+web-time = "1"
```

Comment:
I believe pinning to the minor value is more accepted than the major

Reply:
Could you elaborate? `web-time = "1"` [means](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio) any version `x` such that `1.0.0 <= x < 2.0.0`, so I wouldn't call it *pinned*.

Reply:
Was just trying to be helpful but I'll leave it up to the maintainers. If you look around think you'll notice its more common to pin to the minor version (1.1 instead of 1) than the major version out of an abundance of caution. Even open source maintainers can make mistakes and ship breaking changes on accident under minor version bumps.

Additionally, I think dependency version bumps unrelated to feature changes could go in separate PRs. 

I'm stoked to see work towards floem supporting the web target!

Reply:
Thanks for the feedback! Note that `web-time = "1.1"` means `1.1 <= x < 2.0.0` and **not** `1.1 <= x < 1.2`. So this is needed when a feature you rely on has been added in a minor version update. `web-time = "1"` is equivalent to ` web-time = "1.0"`.
Nevertheless, I agree that one needs to be careful with specifying only major versions by making sure no feature added in a later minor release has accidentally been used, so always specifying the latest `major.minor` as a lower bound is probably a good habit.

Reply:
Haha yep, I just looked at web-time on crates.io because I hadn't seen it before and noticed it was on 1.1 already